### PR TITLE
Fix ZET typo to conform to naming convention.

### LINKED ISSUE #157: `ZET_STRUCTURE_TYPE_METRICS_CALCULATE_EXP_DESC` does not respect naming convention
`ZET_STRUCTURE_TYPE_METRICS_CALCULATE_EXP_DESC` should be `ZET_STRUCTURE_TYPE_METRIC_CALCULATE_EXP_DESC` to match it's structure name `zet_metric_calculate_exp_desc_t`.

### LINKED ISSUE #158: `ZET_STRUCTURE_TYPE_GLOBAL_METRICS_TIMESTAMPS_EXP_PROPERTIES` does not respect naming convention
`ZET_STRUCTURE_TYPE_GLOBAL_METRICS_TIMESTAMPS_EXP_PROPERTIES` should be named `ZET_STRUCTURE_TYPE_METRIC_GLOBAL_TIMESTAMPS_RESOLUTION_EXP` to match it's structure name `zet_metric_global_timestamps_resolution_exp_t`.

### PR DESCRIPTION
**Pull request by wdamon-intel**
- Add ZET_STRUCTURE_TYPE_METRIC_CALCULATE_EXP_DESC
- Add ZET_STRUCTURE_TYPE_METRIC_GLOBAL_TIMESTAMPS_RESOLUTION_EXP
- Deprecate ZET_STRUCTURE_TYPE_METRICS_CALCULATE_EXP_DESC
- Deprecate ZET_STRUCTURE_TYPE_GLOBAL_METRICS_TIMESTAMPS_EXP_PROPERTIES

Resolves #157 
Resolves #158

### CODE REVIEW

**Comment by wdamon-intel** [comment]
@jandres742 please review.

File: scripts/tools/common.yml [hunk 1]
```diff
@@ -109,8 +109,15 @@ etors:
       desc: $t_tracer_exp_desc_t
       value: "0x00010001"
     - name: METRICS_CALCULATE_EXP_DESC
-      desc: $t_metric_calculate_exp_desc_t
+      desc:
```
Comment (jandres742) [inline scripts/tools/common.yml:112]: we have other places with also mixed names, like above:

GLOBAL_METRICS_TIMESTAMPS_EXP_PROPERTIES

would we want to change it on this PR or another one?

**Review by jandres742 — APPROVED** [review]

File: scripts/tools/common.yml (hunk 1 above)
Comment (wdamon-intel) [reply scripts/tools/common.yml:119]: updated

**Comment by wdamon-intel** [comment]
@matcabral please review.

**Comment by matcabral** [comment]
@wdamon-intel LGTM

**Pull request merged**